pareto/utilities/results.py

```diff
@@ -334,6 +334,10 @@ def generate_report(model, is_print=[], fname=None):
             "v_B_Production_dict": [
                 ("Pads", "Time", "Produced Water For Transport From Pad")
             ],
+            "v_Q_dict": [("Location", "Water Component", "Time", "Water Quality")],
+            "v_F_UnusedTreatedWater_dict": [
```

Comment:
In PR description this variable is still called v_F_Treatment_Waste_dict. Also in the PR description it is stated it will output PARETO_report_water_quality.xlsx, but I don't see that file as output.

Reply:
Great point! The names in the code are most up to date. Additionally, we have since simplified output to a singular output file. These changes occurred since I wrote the description. 
I have updated the PR description to reflect these accurately.